src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -12,6 +12,9 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeModeOptions, Geocoder
     // Number of millis to wait on input changes before sending directions request
     var DIRECTION_THROTTLE_MILLIS = 750;
 
+    // default maxWalk when biking
+    var MAXWALK_BIKE = 300;
```

Comment:
Default maxWalkDistance is greater; it is 800.  This is only .19 miles, right? Also would be good to rename constant and/or add note regarding units/what this is in miles.